homeassistant/components/shelly/__init__.py

```diff
@@ -320,6 +330,41 @@ async def _async_setup_rpc_entry(hass: HomeAssistant, entry: ShellyConfigEntry)
         await hass.config_entries.async_forward_entry_setups(
             entry, runtime_data.platforms
         )
+
+        # Latest available firmware for Plug S Gen3 and Outdoor Plug S Gen3 is 1.2.3.
```

Comment:
Would be cleaner to put this code in a named function so its easier to remove later when this firmware is long old